listeners.go

```diff
@@ -39,28 +40,31 @@ type krazyServer struct {
 
 var (
 	listenersMu sync.Mutex
-	listeners   = make(map[string][]*krazyServer) // by private IP address or unix socket path
+	listeners   = make(map[string][]*krazyServer) // by unix socket path and private IP address
 )
 
-// gokrazyHTTPUnixSocket is the unix socket path that
-// the HTTP server listens on when httpPassword is empty.
-const gokrazyHTTPUnixSocket = "/run/gokrazy-http.sock"
+// GokrazyHTTPUnixSocket is the unix socket path that the HTTP server listens on.
+const GokrazyHTTPUnixSocket = "/run/gokrazy-http.sock"
 
 // tlsConfig: tlsConfig. nil, if the listeners should not use https (e.g. for redirects)
 func updateListeners(port, redirectPort string, tlsEnabled bool, tlsConfig *tls.Config) error {
-	var hosts []string // private IP addresses or unix socket path
-	var err error
-
+	// Always listen on a Unix socket.
+	//
+	// Local packages running as root can connect through the Unix socket
+	// regardless of TLS settings. With TLS enabled: Avoids the complexity
```

Comment:
I’m not sure the TLS settings deserve this much mention in the comment. How about:

```
// Local processes running as root can connect through the Unix socket,
// which removes the complexity of connecting via HTTP+TCP
// (which might need a custom port, or TLS credentials).
```